packages/block-renderer/src/hooks/use-pattern-min-height-vh.ts

```diff
@@ -0,0 +1,21 @@
+import { useMemo } from 'react';
+
+const usePatternMinHeightVh = ( html = '' ) => {
+	return useMemo( () => {
+		const minHeightVhResults = html.match( /min-height:(\d+)vh;?/ );
```

Comment:
The part `;?` is because maybe there are more inlined styles so we match both `min-height:75vh` or `min-height:75vh;`.

<img width="227" alt="Screenshot 2566-08-30 at 16 55 08" src="https://github.com/Automattic/wp-calypso/assets/1881481/8dba0b2d-cbc9-4263-8704-11805c677be5">

Reply:
It might be better to replace all inline styles with the min-height. Also, there might be spaces between the property name and its value. How about the following one

```js
return html.replace(/min-height:\s+(?<value>\d+)vh;?/g, ( match, value ) => `min-height:${Number(value) * viewportHeight}px;`);
```

See [Specifying a function as the replacement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#specifying_a_function_as_the_replacement).